### ISSUE TITLE
consul: Await Tailscale service

### ISSUE THREAD

**Issue by VoyTechnology** [VoyTechnology/home-ansible#1]
Before we start the consul service, we should wait for the tailscale service to be enabled, as we require the `tailscale0` network interface to be up.

**Comment by VoyTechnology** [comment]
Consul will most likely be deployed in k8s, if even needed

**Comment by VoyTechnology** [comment]
No longer relevant
